.travis.yml

```diff
@@ -25,13 +25,6 @@ script:
   - GOOS=windows GOARCH=amd64 BINSUFFIX=-windows-amd64.exe make
   - GOOS=windows GOARCH=386 BINSUFFIX=-windows-386.exe make
   - GOOS=darwin GOARCH=amd64 BINSUFFIX=-darwin-amd64 make
-  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-  - "[ ! -z \"$TRAVIS_TAG\" ] && make docker || echo 'non-tag build'"
 
 before_install:
   - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-  - curl -L https://github.com/tcnksm/ghr/releases/download/v0.12.0/ghr_v0.12.0_linux_amd64.tar.gz --output ghr.tar.gz && tar zxvf ghr.tar.gz
```

Comment:
Thanks for the PR. Why did you drop building of non-docker binaries and uploading them to GitHub Releases?

Reply:
Hi @mlapshin, this step would always fail for someone contributing from the outside no? 
As they would not have permission to push new releases to your project (nor should they ever have).